### DISCUSSION TITLE
cant access settings

### DISCUSSION THREAD

**Discussion by Laxdal** [MicaForEveryone/MicaForEveryone#77] [General]
I tried to install this amazing software, but I could not open the setup page and the software would not start either the .Zip version or the msix version where you have to install the license. I also installed .Net core and restarted my computer several times. When I click on the exe file, the icon shows up in the system tray, but when I click on the icon, a window pop up for like 2 seconds then disappears. please help.

im using windows 10

**Comment by dongle-the-gadget** [comment]
It's a known bug with Windows 10. See #65

**Reply by Laxdal** [reply]
got it, thanks!

**Reply by xmine64** [reply]
But settings window should get open, doesn't it?

EDIT: I got it it's not released yet
